### ISSUE TITLE
Homebrew tap marnen/extras fails because sha1 isn't supported

### ISSUE THREAD

**Issue by heinsdorf** [marnen/homebrew-extras#1]
I was trying to install frescobaldi onto Mac OS Sierra 10.12.2

~/Documents/src/python/lilypond $ brew tap marnen/extras
==> Tapping marnen/extras
Cloning into '/usr/local/Homebrew/Library/Taps/marnen/homebrew-extras'...
remote: Counting objects: 5, done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 5 (delta 0), reused 3 (delta 0), pack-reused 0
Unpacking objects: 100% (5/5), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/marnen/homebrew-extras/frescobaldi.rb
Calling Formula.sha1 is disabled!
**Use Formula.sha256 instead.**
/usr/local/Homebrew/Library/Taps/marnen/homebrew-extras/frescobaldi.rb:6:in `<class:Frescobaldi>'
Please report this to the marnen/extras tap!
Error: Cannot tap marnen/extras: invalid syntax in tap!

**Comment by marnen** [comment]
I don't recommend installing Frescobaldi through Homebrew anymore; this was a stopgap measure until actual Mac releases became available. I now recommend installing Frescobaldi from one of the Mac installers available at http://www.frescobaldi.org/download.

However, you're right that this formula should not be broken. Hopefully it's only a question of replacing `sha1` with `sha256`...let me check.

**Comment by marnen** [comment]
This should be fixed in 428e11f (and I updated the version of Frescobaldi in the formula too). However, as I said above, please don't install Frescobaldi through Homebrew. :)

**Comment by heinsdorf** [comment]
Thanks for your reply. You would also need to change the sha256 literal strings to match the file checksums in github.

**Comment by marnen** [comment]
Well, of course. :) I already did that.

**Comment by heinsdorf** [comment]
Thanks, I had already done the install directly from github as you suggested. I just wanted to save other people from wasting time on that bad brew.

**Comment by marnen** [comment]
I appreciate your letting me know about this. Thanks!

**Comment by heinsdorf** [comment]
And I appreciate your effort in making the homebrew extras available to the public. Homebrew is more convenient than github. In the end, I decided to use MuseScore instead of Frescobaldi, because MuseScore doesn’t require any dependencies.

**Comment by marnen** [comment]
Frescobaldi.app doesn't require any dependencies either (other than LilyPond, which is also available as a self-contained app bundle). And Lilypond/Frescobaldi produces better notation than MuseScore.

**Comment by heinsdorf** [comment]
I didn’t have the required midi output installed, and had some problem installing portmidi. MuseScore comes with its own audio output. I’ve used Lillypond, and I agree that it produces excellent printed scores. But MuseScore is “good enough for jazz”, and that’s what I”m using it for. BTW, I’m on Mac OS Sierra.

**Comment by marnen** [comment]
You don't need portmidi for current versions of Frescobaldi.app; the app bundle is completely self-contained. (I use SimpleSynth to receive the MIDI output from Fresco; that's also a self-contained app bundle. But you could also send it to an external synth, or anywhere else.)

**Comment by heinsdorf** [comment]
I also had some problem with SimpleSynth. I forget what — might have been something about music fonts. I never knew that midi used fonts. I have an old cheap midi keyboard, but I doubt that I have the right connectors for it. Anyway, I have something that works for me now, and as the saying goes, Perfect is the enemy of the Good.

**Comment by marnen** [comment]
MIDI doesn't use fonts. SimpleSynth can play soundfonts, but you can also use it without. It has always worked flawlessly to me. 

Are you sure it was SimpleSynth you tried, not fluidsynth (which is less straightforward on Mac OS)?

**Comment by marnen** [comment]
And the saying goes that the good is the enemy of the better. Fresco was once much harder to install on Mac OS, but those days are long gone. So try the better again when you get a chance. :) E-mail me directly if you need help, or try the mailing list.

**Comment by heinsdorf** [comment]
Ahh, you’re right. I was trying to install Denemo, which uses FluidSynth. After reaching an impasse on that, I looked at SimpleSynth, but was immediately scared away when I saw that it hadn’t been updated in seven years. Since updating my Mac to Sierra, I’ve had several incompatibilities with various applications, and wasn’t eager to find another one.

Following your suggestion, I installed SimpleSynth this morning, and after I realized that the score needed a \midi {} command, AND that the “play” button would work, even though it was grayed out, I began to hear the output for the first time.

When I tried to use the xml2ly converter to import my MuseScore files, I got a traceback. Well, 2 out of 3 ain’t bad. Anyway, I’ll now switch to Frescobaldi, because I am already familiar with LilyPond and much prefer typing notes instead of using MuseScore's GUI. The examples I saw in Frescobaldi looked cool. 

Thanks for your help.
